menderqa_pipeline.go

```diff
@@ -441,30 +441,58 @@ func getBuildParameters(
 
 	// set the rest of the CI build parameters
 	runIntegrationTests := "true"
+	runBackendIntegrationTests := "true"
```

Comment:
Are we adding support for this thing now?

Reply:
Not generic support, no. It's just to omit these builds from the C++ client builds.